content/docs/0.8.x/integrations/action_provider/index.md

```diff
@@ -43,60 +55,60 @@ The [Keptn CloudEvents](#cloudevents) an action-provider has to subscribe to is:
 
-3. **Send start event:** If the action is supported, send a start event of type: [sh.keptn.event.action.started](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#action-started). This CloudEvent informs Keptn that a service takes care of executing the action. 
+3. **Send a started event:** If the action is supported, send a start event of type: `sh.keptn.event.action.started`. This CloudEvent informs Keptn that a service takes care of executing the action. 
 
 4. Execute the implemented functionality. At this step, the action-provider can make use of another automation tool. 
 
-5. **Send finished event:** Send a finished event of type: [sh.keptn.event.action.finished](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#action-finished). This informs Keptn to proceed in the remediation or operational workflow. 
-
-**Deployment and service template:** Like any custom *Keptn-service*, an action-provider is a regular Kubernetes service with a deployment and service template. See [here](../custom_integration/#example-jmeter-service) how to define those templates for your action-provider. 
-
-## Subscribe Action-provider to Keptn event
-
-**Distributor:** To subscribe your action-provider to the `sh.keptn.event.action.triggered` event, a distributor is required. A distributor comes with a deployment manifest as shown by the example below:
-
-```yaml
-## action-provider: sh.keptn.event.action.triggered
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: action-provider-distributor
-  namespace: keptn
-spec:
-  selector:
-    matchLabels:
-      run: distributor
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        run: distributor
-    spec:
-      containers:
-      - name: distributor
-        image: keptn/distributor:0.7.3
-        ports:
-        - containerPort: 8080
-        resources:
-          requests:
-            memory: "32Mi"
-            cpu: "50m"
-          limits:
-            memory: "128Mi"
-            cpu: "500m"
-        env:
-        - name: PUBSUB_URL
-          value: 'nats://keptn-nats-cluster'
-        - name: PUBSUB_TOPIC
-          value: 'sh.keptn.event.action.triggered'
-        - name: PUBSUB_RECIPIENT
-          value: 'action-provider'
+5. **Send a finished event:** Send a finished event of type: `sh.keptn.event.action.finished`. The data block of the event payload must provide at least a value for `status` and `result` in `data.action`:
+
+  * `status`: [succeeded, errored, unknown] - The status of the task execution
+  * `result`: [pass, failed] - The result of a successful task execution 
+
+```json
+{
+  "type": "sh.keptn.event.action.finished",
+  "specversion": "1.0",
+  "source": "https://github.com/keptn/keptn/unleash-service",
+  "id": "ggb878d3-03c0-4e8f-bc3f-454bc1b3d888",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext": "08735340-6f9e-4b32-97ff-3b6c292bc509",
+  "triggeredid": "2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
```

Comment:
let's add a short description on how to set the triggeredid

Reply:
added a description now